wagtail_localize/machine_translators/tests/test_libretranslate_translator.py

```diff
@@ -0,0 +1,109 @@
+from unittest import mock
+
+from django.test import TestCase, override_settings
+from wagtail.models import Locale
+
+from wagtail_localize.machine_translators import get_machine_translator
+from wagtail_localize.machine_translators.libretranslate import LibreTranslator
+from wagtail_localize.strings import StringValue
+
+
+LIBRETRANSLATE_SETTINGS_ENDPOINT = {
+    "CLASS": "wagtail_localize.machine_translators.libretranslate.LibreTranslator",
+    "OPTIONS": {
+        "LIBRETRANSLATE_URL": "https://libretranslate.org",
+        "API_KEY": "test-api-key",
+    },
+}
+
+
+class TestLibreTranslator(TestCase):
+    @override_settings(
+        WAGTAILLOCALIZE_MACHINE_TRANSLATOR=LIBRETRANSLATE_SETTINGS_ENDPOINT
+    )
+    def setUp(self):
+        self.english_locale = Locale.objects.get()
+        self.french_locale = Locale.objects.create(language_code="fr-fr")
+        self.translator = get_machine_translator()
+
+    def test_api_endpoint(self):
+        self.assertIsInstance(self.translator, LibreTranslator)
+        api_endpoint = self.translator.get_api_endpoint()
+        self.assertEqual(api_endpoint, "https://libretranslate.org")
+
+    @mock.patch(
```

Comment:
this isn't quite right. please don't mock the `translate` result, but the result of `requests.post()` - i.e. mock the API requests to you can test what `self.translator.translate()` gives you